src/renderer/assets/themes/prismjs/dark.theme.css

```diff
@@ -109,7 +108,8 @@ pre.ag-paragraph {
 }
  
 .token.regex,
-.token.important {
+.token.important,
+.token.variable {
```

Comment:
Can you show me an example when will use `token.variable` style, or screenshot to show the `before` and `after` style.